openstack/compute/v2/extensions/hypervisors/results.go

```diff
@@ -123,6 +144,12 @@ type Hypervisor struct {
 	// Service is the service this hypervisor represents.
 	Service Service `json:"service"`
 
+	// Servers is a list of Server object.
+	// The requires microversion 2.53 or later.
+	// This field has become mandatory in microversion 2.75.
+	// If no servers is on hypervisor then empty list is returned.
+	Servers []Server `json:"servers"`
```

Comment:
Please change this to `*[]Server`. The reason is described in our [microversions](https://github.com/gophercloud/gophercloud/blob/master/docs/MICROVERSIONS.md#new-response-fields) document, but let me know if you have any questions.

A similar example of using `*[]` is here: https://github.com/gophercloud/gophercloud/blob/841705fad1df2f50b23c31ee60f51702063be7c6/openstack/compute/v2/extensions/instanceactions/results.go#L141-L143